docs/sphinx-docs/doc-regen/makedocumentation.py

```diff
@@ -0,0 +1,132 @@
+"""
+Creates documentation from .py files
+"""
+import os
+import sys
+from os.path import join, abspath, dirname, basename
+import subprocess
+
+FILE_ABS_SOURCE = abspath(dirname(__file__))
+MAIN_PY_SRC = FILE_ABS_SOURCE + "/../source-temp/user/models/src/"
+if os.path.exists(MAIN_PY_SRC):
+    ABSOLUTE_TARGET_MAIN = abspath(join(dirname(__file__), MAIN_PY_SRC))
+    PLUGIN_PY_SRC = abspath(dirname(__file__)) + "/../../../../.sasview/plugin_models/"
```

Comment:
Many of the pathing issues could be overcome by copying all documentation files into the user directory on installation. Finding the user directory is trivial (see `find_plugin_models()` as an example).
```
Example structure:
~/.sasview/
    docs/
        original/  # The unedited files in case the user wants to blow away their changes (future work...)
        modified/ # Any modified files - load from here, if the file exists, otherwise grab the original.
```

Reply:
Implemented Paul's idea. I think Jeff's idea is a good idea for refactoring the code but would take a bit of work at the moment.